### DISCUSSION TITLE
error : AttributeError: module 'torch.library' has no attribute 'custom_op'

### DISCUSSION THREAD

**Discussion by 1272870698** [huggingface/diffusers#9725] [Q&A]
code : 
![image](https://github.com/user-attachments/assets/f96296b8-90af-4b38-9055-b210742dfce9)

error : 

![image](https://github.com/user-attachments/assets/6a6d0fb2-2702-487e-b03f-594422f0e9cc)

python lib : 
![image](https://github.com/user-attachments/assets/22ed342e-7251-4993-91ad-95393fa31598)

please  help me ！！ ！

**Comment by tolgacangoz** [comment]
[It seems that (at least current version of) `optimum-quanto` requires `torch>=2.4.0`.](https://github.com/huggingface/optimum-quanto/blob/852bb9cb6fb707a6fcebff7e068dc6bbdda779cb/pyproject.toml#L22)
